Xamarin.Essentials/HapticFeedback/HapticFeedback.shared.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Xamarin.Essentials
+{
+    public static partial class HapticFeedback
+    {
+        public static void Execute(HapticFeedbackType type = HapticFeedbackType.Click)
```

Comment:
Looking at the API s it stands, there seems to be an extra layer of "work"... enum to methods back to enum. I would just use the enum version and iOS can use a switch or split it up that side.

Reply:
I'll think